lib/ansible/modules/cloud/amazon/s3.py

```diff
@@ -129,10 +132,9 @@
     version_added: "1.2"
   region:
     description:
-     - >
-       AWS region to create the bucket in. If not set then the value of the AWS_REGION and EC2_REGION environment variables are checked,
-       followed by the aws_region and ec2_region settings in the Boto config file.  If none of those are set the region defaults to the
-       S3 Location: US Standard.  Prior to ansible 1.8 this parameter could be specified but had no effect.
+     - "AWS region to create the bucket in. If not set then the value of the AWS_REGION and EC2_REGION environment variables
+       are checked, followed by the aws_region and ec2_region settings in the Boto config file. If none of those are set the
+       region defaults to the S3 Location: US Standard. Prior to ansible 1.8 this parameter could be specified but had no effect."
     required: false
```

Comment:
Could the region be a required parameter? It seems like Location.DEFAULT is really vestigial after switching to boto3. Looks like there was an old issue in ansible-modules-core (https://github.com/ansible/ansible-modules-core/issues/964#issuecomment-86552229) where the Location.DEFAULT causing a NoneType was discussed. If it must stay as not required, would it make sense to hardcode this to us-east-1?